app/Http/Controllers/RoomController.php

```diff
@@ -142,6 +144,10 @@ public function updateRoom(Request $request, int $id) {
     private function filterRoomResults(Request $request) {
```

Comment:
Deze functie is best lang, dit kan opgesplitst

Reply:
ben ik wel mee eens, ik vind het een stuk minder leesbaar dan eerst.
zou je je date of arrival & departure check in een nieuwe functie kunnen zetten, en het resultaat hiervan direct returnen.